patches/api/0418-Bell-API.patch

```diff
@@ -0,0 +1,65 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Lama06 <andreasprues36@gmail.com>
+Date: Sun, 22 Jan 2023 11:22:49 +0100
+Subject: [PATCH] Bell API
+
+
+diff --git a/src/main/java/org/bukkit/block/Bell.java b/src/main/java/org/bukkit/block/Bell.java
+index 6fee356a3fc2811d99d9ed2a551118e3e6f0de79..d077bab39dd702b5280455b33db417be4eef6429 100644
+--- a/src/main/java/org/bukkit/block/Bell.java
++++ b/src/main/java/org/bukkit/block/Bell.java
+@@ -3,4 +3,53 @@ package org.bukkit.block;
+ /**
+  * Represents a captured state of Bell.
+  */
+-public interface Bell extends TileState { }
++public interface Bell extends TileState {
++    // Paper start - Bell API
++    /**
++     * Rings the bell in the direction that it is facing.
++     */
++    void ring();
```

Comment:
This method also requires more documentation imo detailing something that, at least `ring` does imply.
The lack of sound when calling this method.

The same goes for the below method, neither will cause a sound to be played.
While this is "understandable" for developers deep in the server code, it should be documented to avoid confusion.

Reply:
I now mention this in the javadocs of this method.
Thanks for all your feedback!